client/src/components/Chat/Messages/Content/ToolCall.tsx

```diff
@@ -134,6 +137,28 @@ export default function ToolCall({
             function_name={function_name}
           />
         )}
+
+        {/*TODO: fix hardcoded action_id*/}
+        {needsLogin && 'Ty2HxVutaMeSDbxkryip6' && (
+          <div className="pl-7 flex flex-col">
+            <div className="text-sm text-gray-600 mb-1">
+              {/* Here we use the original domain value */}
+              {localize('com_assistants_wants_to_talk', domain ?? '' )}
+            </div>
+            <a
+              className="inline-flex w-fit items-center justify-center rounded-full bg-black px-4 py-2 text-sm font-medium text-white hover:bg-gray-800"
+              href={'/api/actions/Ty2HxVutaMeSDbxkryip6/oauth/login'}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {/* And here as well */}
+              {localize('com_assistants_sign_in_with_domain', domain ?? '')}
+            </a>
+            <p className="mt-1 text-xs text-gray-500">
+              {localize('com_assistants_allow_sites_you_trust')}
```

Comment:
i need to figure out how i get obtain the domain name normaly to show for the login button and the `action_id` because this is now a hardcoded value. + plus a feedback for the user what the url will be for the callback action.